src/Stripe.net/Services/Charges/StripeChargeLevel3Options.cs

```diff
@@ -0,0 +1,25 @@
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Stripe
+{
+    public class StripeChargeLevel3Options : INestedOptions
+    {
+        [JsonProperty("level3[merchant_reference]")]
```

Comment:
This works but since the parent has `JsonProperty(level3)` why isn't it passed to the lower level and you have to repeat it? The same happened to shipping.